portfolio/src/main/webapp/index.html

```diff
@@ -6,14 +6,235 @@
+        <br>
+        CS 70: Discrete Mathematics and Probability Theory
+        <br>
+        Data 8: The Foundations of Data Science
+        <br>
+        <br>
+        Fall 2021
+        <br>
+        CS 61B: Data Structures
+        <br>
+        EECS 16A: Designing Information Devices and Systems I
+        <br>
+        CS 198: iOS Decal
+        <br>
+        <br>
+        Summer 2021
+        <br>
+        CS 61A: Structure and Interpretation of Computer Programs
+      </p>
+    </section>
+
+    <section class="timeline" id="timeline">
+      <div class="heading white">
+        <h2>Timeline</h2>
+      </div>
+      <div class="container">
+        <div class="timelineList">
+          <ul>
+            <li>
+              <div class="timeline-content">
+                <h3 class="date">Summer 2022</h3>
+                <h1>temp</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur tempora ab laudantium voluptatibus aut eos placeat laborum, quibusdam exercitationem labore.</p>
+              </div>
+            </li>
+            <li>
+              <div class="timeline-content">
+                <h3 class="date">Summer 2022</h3>
+                <h1>CS61B 20-hr uGSI</h1>
+                <p>Teaching Assistant (Undergraduate Student Instructor) for CS61BL (Data Structure) at UC Berkeley. Help with teaching students the course material through leading my own lab, as well as assisting students in office hours.</p>
+              </div>
+            </li>
+            <li>
+              <div class="timeline-content">
+                <h3 class="date">Spring 2022</h3>
+                <h1>CS61B Academic Intern</h1>
+                <p>Lab assistant for CS61B (Data Structure) at UC Berkeley with 1600+ students. Prepare and review course content and supplemental material to support students in lab sections.</p>
+              </div>
+            </li>
+            <li>
+              <div class="timeline-content">
+                <h3 class="date">Fall 2021</h3>
+                <h1>UC Berkeley</h1>
+                <p>Started my undergraduate study and started pursuing a bachelor's in CS.</p>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+     </section>
+
+    <section class="projects" id="projects">
+      <div class="heading black">
+        <h2>Projects</h2>
+        <p>Some projects I have worked on:</p>
+      </div>
+      <div class="content">
+        <div class="project" onclick="location.href='https://cs61c.org/sp22/projects/proj2/';">;
+          <img src= "images/classify.jpg">
+          <h2>Classify</h2>
+          <p>
+            RISC-V assembly code to classify handwritten digits with a simple machine learning algorithm.
+          </p>
+        </div>
+          <div class="project" onclick="location.href='https://cs61c.org/sp22/projects/proj3/';">
+            <img src= "images/cpu.jpg">
```

Comment:
Include alt tags for multimedia tags for accessibility.
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback